### DISCUSSION TITLE
edge_tts Model

### DISCUSSION THREAD

**Discussion by jhoanmartinez** [rany2/edge-tts#153] [General]
Do you guys know which model is that one? or if the model is open source? I have search the same one on hugging face and github but not results, edge tts is pretty good, voice, languages and regions, like english usa and uk, thank you.

**Comment by rany2** [comment]
It's not open source and you won't find the model for it. It uses an online service that the Edge Browser uses for the read aloud feature. If you'd like the paid/commercial option, Microsoft calls it Azure Cognitive Services and they have a free tier of it. Everything this library accomplishes along with the voices are found on Azure with more advanced features.
